ansible/roles/ocp4-workload-security-compliance-lab/tasks/quay.yml

```diff
@@ -9,6 +9,12 @@
     dest: "{{ tmp_dir }}/quay-operator"
     version: "4ba1d1620e92776dd40decad9e8d2cc5ac49df7f"
 
+- name: Update operator version
+  replace:
+    path: "{{ tmp_dir }}/quay-operator/deploy/operator.yaml"
+    regexp: "image: quay.io/redhat-cop/quay-operator:latest"
+    replace: "image: quay.io/redhat-cop/quay-operator:v0.0.1"
```

Comment:
Maybe we can ask Andrew to add a parameter to overwrite `latest` ?

Reply:
@fridim right now this is pretty short turnaround - we're actually checking out Andrew's git repo and switching to a specific commit (pre-summit). Moving forward, we can probably work with him on maybe creating a new branch from this commit to support the older version of Quay with a dynamic version (or just updated to point to this version)